### ISSUE TITLE
Unable to log in

### ISSUE THREAD

**Issue by PoempelPinguin** [papermerge/papermerge-core#134]
Followed setup instructions given at https://docs.papermerge.io/Setup/docker-compose.html
Works out fine till login, says: "SyntaxError: Unexpected token '<', " <!doctype "... is not valid JSON"

![papermerge_login_failed](https://user-images.githubusercontent.com/25886752/227379646-10a2b196-160c-4716-ac15-a345954c636a.png)

What can I do? Google ist not really helpful since I am a noob :(

**Comment by ciur** [comment]
What version of docker image are you using? Is it 2.1.8?

**Comment by PoempelPinguin** [comment]
After having some other issues I tried a fresh installation.
For docker I followed the exact "Install using apt repository" https://docs.docker.com/engine/install/debian/
So I think ist the latest version. 

I manually copy&pasted docker-compose.yml contents , so yes, I am sure. 😀

Just changed some ,env content (DB User, DB Password, DB Name, secret key and superuser credentials.

**Comment by PoempelPinguin** [comment]
Why did you ask? Issue persists...

**Comment by dan3o3** [comment]
I have the exact same problem here with a fresh install.
I used instructions from https://docs.papermerge.io/Installation/portainer.html so the actual yml from Github is used.

`root@docker:~# docker -v
Docker version 23.0.1, build a5ee5b1`

Any idea how to proceed?

**Comment by VadarVillage** [comment]
I'm having the same issue on 2.1.9, I followed the default guide. I have a similiar error message as the one in the screenshot

**Comment by VadarVillage** [comment]
I had the same issue yesterday, so I double checked my docker-compose.yaml file today and that was the issue.

The default file in the instructions here: https://docs.papermerge.io/Setup/docker-compose.html - assumes that the directories exist on the host OS.  I am new to docker so I was not able to immediately determine if the default file needs to be modified, if those file paths referred to paths inside of the docker container, or if I needed to change them to suit my environment.

After setting up the host OS with the directories that I intend to use for those purposes, I changed the values in the docker compose file and I can now log in.  I am seeing a django error when I start the container, i'm not sure if this is related to the configuration:

- ERROR/ForkPoolWorker-2] Task papermerge.search.tasks.update_index[d7ec70fe-0ab7-4297-953f-334abca3ef2a] raised unexpected: OperationalError('no such table: core_folder')

- django.db.utils.OperationalError: no such table: core_folder
